cogfiles/web_scrapers.py

```diff
@@ -116,25 +120,20 @@ async def heady(self, context: commands.Context, *, song_name: str):
 
         embed = RandomColorEmbed(title=title, url=response.url, description=description)
         embed.set_thumbnail(url="https://clipartspub.com/images/grateful-dead-clipart-template-5.png")
-        await context.send(embed=embed)
-
-    @commands.command()
-    async def anilist(self, context: commands.Context, *, search: str):
-        """Search AniList for an anime or manga with a title matching the provided search."""
-        await WebScrapers.anilist_search(context, search, anime=True, manga=True)
+        await interaction.followup.send(embed=embed)
 
-    @commands.command()
-    async def anime(self, context: commands.Context, *, search: str):
+    @app_commands.command()
+    async def anime(self, interaction: discord.Interaction, search: str):
         """Search AniList for an anime with a title matching the provided search."""
-        await WebScrapers.anilist_search(context, search, anime=True)
+        await WebScrapers.anilist_search(interaction, search, anime=True)
         
-    @commands.command()
-    async def manga(self, context: commands.Context, *, search: str):
+    @app_commands.command()
+    async def manga(self, interaction: discord.Interaction, search: str):
         """Search AniList for a manga with a title matching the provided search."""       
-        await WebScrapers.anilist_search(context, search, manga=True)
+        await WebScrapers.anilist_search(interaction, search, manga=True)

```

Comment:
I wonder if it might make more sense to have one `anilist` command with an enum select for "Anime" or "Manga" search type, instead of two separate commands... up to you.